fre/pp/ppval_script.py

```diff
@@ -10,11 +10,29 @@
 fre_logger = logging.getLogger(__name__)
 
 # Get estimated number of timesteps
-def getenot(date_start,date_end,chunk_type,cal):
+def getenot(date_start: str, date_end:str, chunk_type:str, cal: str):
+    """
+
+    Returns the estimated number of timesteps using elapsed time (calculated using date_start/date_end) and data frequency (provided in chunk_type argument).
+    Date string formats must be YYYY,YYYYMM,YYYYMMDD,YYYYMMDDHH,or YYYYMMDDHH:mm
```

Comment:
Can you add these to https://github.com/NOAA-GFDL/fre-cli/blob/main/.cspell/ok-unknown-words.txt